Lib/concurrent/futures/process.py

```diff
@@ -855,3 +863,66 @@ def shutdown(self, wait=True, *, cancel_futures=False):
         self._executor_manager_thread_wakeup = None
 
     shutdown.__doc__ = _base.Executor.shutdown.__doc__
+
+    def _terminate_or_kill_workers(self, operation):
```

Comment:
```suggestion
    def _force_shutdown(self, operation):
```

How about this name? because we could say that we can generalize it later with other callback operation.

Reply:
Perfect. I had a lot of trouble coming up with a better sound name. Thanks!